test/integration/targets/apt/tasks/main.yml

```diff
@@ -23,6 +23,7 @@
   - import_tasks: 'apt-multiarch.yml'
     when:
       - ansible_userspace_architecture != apt_foreign_arch
+      - not (ansible_distribution == 'Ubuntu' and ansible_distribution_version == '20.04')
```

Comment:
Can you include a comment explaining why this is being done?

Reply:
I'm not entirely sure this line is even necessary yet, I just wanted to see what else failed.

I'm going to try replacing the tested package with a different one and see if it works.